generator.yaml

```diff
@@ -494,11 +494,6 @@ resources:
         from:
           operation: DescribeModelPackage
           path: ModelPackageStatusDetails
-      ModelApprovalStatus:
```

Comment:
something wrong here, it undoes the late initializer stuff?

Reply:
Opened pr before late_initialize got merged, will rebase